python/tk_alias/menu_generation.py

```diff
@@ -188,22 +188,40 @@ def _jump_to_fs(self):
             if exit_code != 0:
                 self._engine.logger.error("Failed to launch '%s'!", cmd)
 
-    def _version_check(self, version):
+    def _version_check(self, version1, version2):
         """
         Check if Alias version is greater than 2022.2 for ShotGrid menu name
 
-        :param version: String value of Alias version from startup.py
+        :param version1: String value of Alias version from startup.py
+        :param version2: String value of comparison version from this file
```

Comment:
These version strings do not specifically _need_ to be the Alis version from startup.py or the version from this file, the caller can pass in any version strings that they want to compare. So we should just list them as:
```
:param version1: A version string to compare against version2.
:param version2: A version string to compare against version1.
```